src/plots/polar/polar.js

```diff
@@ -597,6 +597,9 @@ proto.updateAngularAxis = function(fullLayout, polarLayout) {
     }
     _this.vangles = vangles;
 
+    // TODO maybe two arcs is better here?
+    // maybe split style attributes between inner and outer angular axes?
```

Comment:
If we can do what we need right now without this, we can add it in later. But it opens up a can of worms if you combine hole and sector... do you allow separate styling of the radial segments? and joining these paths cleanly where they meet is a headache - it was painful enough to get that right for cartesian https://rreusser.github.io/plotly-mock-viewer/#20 but here you also need to deal with weird angles and curves... I don't even know in principle how to handle that!

Reply:
Leaving this for _later_, if ok.

Reply:
-> https://github.com/plotly/plotly.js/issues/2255#issuecomment-419575450